### PR TITLE
Remove obsolete terminology from documentation

### PR DESCRIPTION
**Pull request by alcaeus**
Ping @jmikola

### CODE REVIEW

File: reference/mongodb/mongodb/driver/query/construct.xml [hunk 1]
```diff
@@ -368,11 +368,6 @@
           <entry><type>int</type></entry>
           <entry>Number of documents to skip. Defaults to 0.</entry>
          </row>
-         <row>
-          <entry>slaveOk</entry>
-          <entry><type>bool</type></entry>
-          <entry>Allow query of replica set secondaries</entry>
-         </row>
```
Comment (jmikola) [inline reference/mongodb/mongodb/driver/query/construct.xml:375]: This should have been removed when the original changelog entry was added for `ext-mongodb` 1.2.0, correct?

Looking at [this docs page](https://www.php.net/manual/en/mongodb-driver-query.construct.php), it definitely doesn't make sense for it to still appear in this list when we say it was removed in the changelog. We also neglected to add a note about it being deprecated. The relevant PHPC ticket was [PHPC-806](https://jira.mongodb.org/browse/PHPC-806). Docs commit was https://github.com/php/doc-en/commit/0205d3f11d7662c2678523e7e8132e4ac596002d.

**Review by jmikola — APPROVED** [review]
One question but LGTM.

File: reference/mongodb/mongodb/driver/query/construct.xml (hunk 1 above)
Comment (alcaeus) [reply reference/mongodb/mongodb/driver/query/construct.xml:375]: Yes, this has been obsolete for a while.

**Comment by alcaeus** [comment]
Merged in 2de5fb458e.

**Pull request closed without merging**